generics/constants/common.js

```diff
@@ -44,5 +44,8 @@ module.exports = {
     "CHILDREN" : "children",
     "ATTACHMENT_TYPE_LINK" : "link",
     "PROJECT_ATTACHMENT" : "project",
-    "TASK_ATTACHMENT" : "task"
+    "TASK_ATTACHMENT" : "task",
+    "SUNBIRD_SERVER_TIMEOUT" : 5000,
+    "TIMEOUT_ERROR" : "Server timeout error",
```

Comment:
@VISHNUDAS-tunerlabs if its a response status/message  add it in generics/https-status-codes